### ISSUE TITLE
Will this work in SSMS v. 19

### ISSUE THREAD

**Issue by magnusbratt** [nicholas-ross/SSMS-Schema-Folders#26]
It works beautifylly for me in SSMS v. 18. I tried the same istallation instructions but installed in the folder for SSMS v. 19 (preview 2): `C:\Program Files (x86)\Microsoft SQL Server Management Studio 19\Common7\IDE\Extensions\SSMS Schema Folders`

I unblocked the zip-file, before extracting. 

I answered no to the error message on first start.

When I go to `Tools > Option > SQL Server Object Explorer > Schema folders` I get `An error occured loading this property page`.

**Comment by nicholas-ross** [comment]
Microsoft have added the required assembly binding redirect for 18 to 19. However I am currently defaulting to 2016 for unknown versions.

Attached is a version with the performance tweaks from another issue but it now defaults to version 18. I've tested it works with version 19 preview 2. I will do a proper release when 19 is released. 
[SSMS Schema Folders.zip](https://github.com/nicholas-ross/SSMS-Schema-Folders/files/9036098/SSMS.Schema.Folders.zip)

For those that want to make the change in the code themselves, in file `SsmsSchemaFoldersPackage.cs` function `GetObjectExplorerExtender()` change the final return statement from
```
return new Ssms2016::SsmsSchemaFolders.ObjectExplorerExtender(this, Options);
```
to
```
return new Ssms18::SsmsSchemaFolders.ObjectExplorerExtender(this, Options);
```
